### PR TITLE
Replaced duplicated transaction code with utility method.

### PR DESCRIPTION
**Pull request by ardevd**

### CODE REVIEW

**Comment by tamirms** [comment]
I would prefer it if you defined the helper in the AbstractTransaction class

**Comment by ardevd** [comment]
@tamirms done :)

File: src/main/java/org/stellar/sdk/Util.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import com.google.common.io.BaseEncoding;
 import org.stellar.sdk.xdr.ClaimableBalanceID;
+import org.stellar.sdk.xdr.Hash;
```
Comment (tamirms) [inline src/main/java/org/stellar/sdk/Util.java:5]: I think these are unused imports

Comment (ardevd) [reply src/main/java/org/stellar/sdk/Util.java]: fixed :)

**Review by tamirms — APPROVED** [review]
thank you for your contributions @ardevd !

**Comment by tamirms** [comment]
@ardevd it looks like 2 of your commits are not signed. could you sign them because we cannot merge the PR unless all the commits are signed?

**Comment by ardevd** [comment]
@tamirms I just ended up making a mess. Closing this PR and replacing it with a new one. My apologies.

**Pull request closed without merging**